contributors/guide/pull-requests.md

```diff
@@ -0,0 +1,283 @@
+
+If it's the last few weeks of a milestone, we need to reduce churn and stabilize.
+
+Or, it could be related to best practices. One common issue is that the pull request is too big to review. Let's say you've touched 39 files and have 8657 insertions. When your would-be reviewers pull up the diffs, they run away - this pull request is going to take 4 hours to review and they don't have 4 hours right now. They'll get to it later, just as soon as they have more free time (ha!).
+
+There is a detailed rundown of best practices, including how to avoid too-lengthy pull requests, in the next section.
+
+But, if you've already followed the best practices and you still aren't getting any pull request love, here are some
+things you can do to move the process along:
+
+   * Make sure that your pull request has an assigned reviewer (assignee in GitHub). If not, reply to the pull request comment stream asking for a reviewer to be assigned.
+   <!-- TODO: example -->
+
+   * Ping the assignee (@username) on the pull request comment stream, and ask for an estimate of when they can get to the review.
+
+   * Ping the assignee on [Slack](http://slack.kubernetes.io). Remember that a person's GitHub username might not be the same as their Slack username.
+
+   * Ping the assignee by email (many of us have publicly available email addresses).
+
+   * If you're a member of the organization ping the [team](https://github.com/orgs/kubernetes/teams) (via @team-name) that works in the area you're submitting code.
+
+   * If you have fixed all the issues from a review, and you haven't heard back, you should ping the assignee on the comment stream with a "please take another look" (`PTAL`) or similar comment indicating that you are ready for another review.
+
+Read on to learn more about how to get faster reviews by following best practices.
+
+# Best Practices for Faster Reviews
+
+Most of this section is not specific to Kubernetes, but it's good to keep these best practices in mind when you're making a pull request.
+
+You've just had a brilliant idea on how to make Kubernetes better. Let's call that idea Feature-X. Feature-X is not even that complicated. You have a pretty good idea of how to implement it. You jump in and implement it, fixing a bunch of stuff along the way. You send your pull request - this is awesome! And it sits. And sits. A week goes by and nobody reviews it. Finally, someone offers a few comments, which you fix up and wait for more review. And you wait. Another week or two go by. This is horrible.
+
+Let's talk about best practices so your pull request gets reviewed quickly.
+
+## 0. Familiarize yourself with project conventions
+
+* [Development guide](/contributors/devel/development.md)
+* [Coding conventions](../guide/coding-conventions.md)
+* [API conventions](/contributors/devel/api-conventions.md)
+* [Kubectl conventions](/contributors/devel/kubectl-conventions.md)
+
+## 1. Is the feature wanted? Make a Design Doc or Sketch pull request
```

Comment:
Reference KEP and feature processes.

Looks like a lot has changed since this section was written.